src/components/People/People.tsx

```diff
@@ -0,0 +1,108 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+import { getPeople } from '../../api';
+
+export const People = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
```

Comment:
consider to make initial empty array. it avoid checking on people. just on people length
```suggestion
  const [people, setPeople] = useState<Person[]>([]);
```